app/schemas.py

```diff
@@ -209,7 +209,7 @@ class ServiceSchema(BaseSchema):
     reply_to_email_address = fields.Method(method_name="get_reply_to_email_address")
     sms_sender = fields.Method(method_name="get_sms_sender")
     letter_contact_block = fields.Method(method_name="get_letter_contact")
-    prefix_sms_with_service_name = fields.Method(method_name="get_prefix_sms_with_service_name")
+    prefix_sms = field_for(models.Service, 'prefix_sms')
```

Comment:
the prefix_sms will show up on the schema without this line.

Reply:
Squished out.